hc/api/tests/test_create_check.py

```diff
@@ -35,8 +36,9 @@ def test_it_works(self):
         assert "ping_url" in doc
         self.assertEqual(doc["name"], "Foo")
         self.assertEqual(doc["tags"], "bar,baz")
-
         ### Assert the expected last_ping and n_pings values
+        self.assertIsNone(doc['last_ping'])
```

Comment:
Be consistent whether to use single quotation marks or double quotation marks. 
Choose one consistent convention.
I'd prefer double quotation marks.